salt/modules/tls.py

```diff
@@ -1575,12 +1575,15 @@ def create_pkcs12(ca_name, CN, passphrase='', cacert_path=None, replace=False):
     )
 
 
-def cert_info(cert_path, digest='sha256'):
+def cert_info(cert, digest='sha256'):
```

Comment:
Why are we changing this?

Reply:
`cert` is string|filepath
`cert_path` being a string would be rather confusing

Also it's the first arg which is nowhere used in Salt AFAICT, so pplz using `tls.get_cert_info(myfile)` wont notice a thing.

I can add some deprecation stuff around it if you're bothered by it?